src/pocketmine/permission/BanList.php

```diff
@@ -58,6 +58,19 @@ public function setEnabled(bool $flag){
 		$this->enabled = $flag;
 	}
 
+	/**
+	 * @param string $name
+	 *
+	 * @return BanEntry|null
+	 */
+	public function getEntry(string $name) : ?BanEntry{
+		$name = strtolower($name);
+		
+		$this->removeExpired();
+
+		return $this->list[$name] ?? null;
```

Comment:
the `strtolower()` can now be moved to here, and remove the first statement, since it's only used once.